src/App.js

```diff
@@ -1,13 +1,136 @@
-import React from 'react';
+import React from "react";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+import Search from "./components/SearchComponent/Search";
+import "./App.css";
+
+// Set Task In LocalStorage
+if (localStorage.getItem("tasks") === null) {
+  localStorage.setItem("tasks", JSON.stringify([]));
```

Comment:
Good job using local storage to save data. I believe you should move this logic in your App component though. You probably want to use life cycle methods to achieve that.